app/views/error/access_denied.scala.html

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  *@
 
-@(ggLoginUrl: String, registerOrgUrl: String)(implicit request: Request[_], messages: Messages)
+@(ggLoginUrl: String, registerOrgUrl: String)(implicit request: Request[_], messages: Messages, appConfig: config.AppConfig)
```

Comment:
What does this do?

Reply:
We need it for the gov_uk wrapper to work?

Reply:
The app config contains reference to a few links used across several pages but it's needed everywhere as it also contains the signout link in the govuk wrapper